src/sentry/models/organization.py

```diff
@@ -189,7 +189,7 @@ class flags(TypedClassBitField):
         # Temporarily opt out of new visibility features and ui
         disable_new_visibility_features: bool
 
-        # Require and enforce email verification for all members.
+        # Require and enforce email verification for all members. (deprecated, not in use)
```

Comment:
nit: I don't know how BItField works, and if this would even work, but does it makes sense to make this also a `@property` of a class and add a deprecation warning if someone tries to access this field?

Reply:
Bitfield deserializes the storage integer into a set of flags that quack like booleans, enabling usage like `org.flags.require_email_verification` to be read/written. The `flags` instance is generated via metaclasses so adding deprecations via `@property` will be challenging 🤔